Adding 3x4 OLED demo

BlitzCityDIY opened the pull request:
Adding an example for the 3x4 numpad with an OLED to copy the Arduino lib example (https://github.com/adafruit/Adafruit_TCA8418/blob/main/examples/tca8418_3x4_OLED/tca8418_3x4_OLED.ino)

BlitzCityDIY committed:
Adding 3x4 OLED demo

Adding an example for the 3x4 numpad with an OLED to copy the Arduino lib example

BlitzCityDIY committed:
running pre-commit

Forgot pre-commit 🤦‍♀️

tekktrik commented on examples/tca8418_3x4_OLED.py:
```diff
@@ -0,0 +1,93 @@
+            event = keyevent & 0x7F
+            event -= 1
+            #  figure out row
+            row = int(event / 10)
```
This could be `event // 10` but I'm not sure if that's as beginner friendly as casting with `int()`.  Your call!

BlitzCityDIY replied:
Ohh yeah I like that, thanks! This example is more to show a good example for matrixes so I'd rather have the most pythonic way.

BlitzCityDIY committed:
Changing to event // 10

Ohh yeah I like that, thanks! This example is more to show a good example for matrixes so I'd rather have the most pythonic way.

tekktrik reviewed (approved):
Look good to me!  One small comment but it's not an issue so it's your call or merge as-is!

tekktrik commented:
Well then that settles it!
